rules/linux/execution_shell_via_lolbin_interpreter_linux.toml

```
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2023/07/04"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+This detection rule detects the creation of a shell through a suspicious process chain. Any reverse shells spawned by 
+the specified utilities that are initialized from a single process followed by a network connection attempt will be 
+captured through this rule. Attackers may spawn reverse shells to establish persistence onto a target system.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Reverse Shell Activity via Linux Command Line Utility Detected"
```

too long for a rule.name 

```suggestion
name = "Potential Reverse Shell via Linux Utility"
```